types/src/block_metadata.rs

```diff
@@ -0,0 +1,56 @@
+use crate::account_address::AccountAddress;
+use crypto::{ed25519::Ed25519Signature, HashValue};
+use std::collections::BTreeMap;
+
+/// Struct that will be persisted on chain to store the information of the current block.
```

Comment:
Do you know how this will be stored on chain?

Reply:
Yes! I'll implement the serializer/deserializer shortly. The TL;DR is that it's going to look something similar to the `AcountResource` that we implemented previously. The Move will use a module to operate on the on chain data and I'll implement a ser/de so that we can read it out easily without the VM.